app/views/settings/index.html.erb

```diff
@@ -4,6 +4,9 @@
 ordered_settings = @settings.stick_general_first
 first_category = ordered_settings.values.flatten.first.try(:category)
 %>
+
+<%= react_component('SettingRecords', { :settings => grouped_settings(@settings) }) %>
```

Comment:
Hmm this component has no UI at all right? It's here just to load the settings.
Would it make sense for it to be part of the setting table or if not (I can see why it's easier to do it for all settings at once) should we introduce a general component `SaveMyJsonBlobIntoRedux`?

Reply:
Oh, this would probably get droped after we will manage to get this to a React Page, right?

Reply:
Exactly, I do not think we want to encourage this as a pattern.